src/test/resources/logback-test.xml

```diff
@@ -33,6 +33,7 @@
     <logger name="org.springframework.data.neo4j" level="info"/>
     <logger name="org.springframework.data.neo4j.test" level="info"/>
     <logger name="org.springframework.data.neo4j.repository.query.Neo4jQuerySupport" level="debug" />
+    <logger name="org.springframework.data.neo4j.core.mapping.Neo4jPersistentEntity" level="error" />
```

Comment:
I assume this is due to the old id warnings now, right?

Reply:
Yes